docs/src/AlgebraicGeometry/Schemes/AffineSchemes.md

```diff
@@ -0,0 +1,142 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["AffineSchemes.md"]
+```
+
+
+# Affine schemes
+
+Let ``\mathbb k`` be a commutative noetherian base ring
+(in practice: an algebraic extension of ``\mathbb Q`` or ``\mathbb F_p``).
+We support functionality for affine schemes ``X = Spec(R)`` over ``\mathbb k``.
+Currently, we support rings ``R`` of type `MPolyRing`, `MPolyQuo`,
+`MPolyLocalizedRing`, and `MPolyQuoLocalizedRing`
+defined over the integers or algebraic field extensions of ``\mathbb Q``
+
+
+## Constructors
+
+### General constructors
+
+```@docs
+Spec(R::MPolyRing, I::MPolyIdeal)
+Spec(R::MPolyRing, U::AbsMPolyMultSet)
+Spec(R::MPolyRing, I::MPolyIdeal, U::AbsMPolyMultSet)
```

Comment:
That is an internal constructor and afaik there is no space for a docstring there.

Reply:
@HechtiDerLachs knows his code better than I do. From what I see, there is no constructor (yet) which simply accepts a polynomial ring. This could be added (maybe in a separate PR - this one is already huge). Alternatively, one can use the `affine_space` command to create the affine space. Feature request?

Reply:
Yes, this should go into a separate PR, if we want it. It is one of these compatibility features to make life simpler for the scripting user.